tests/sentry/tasks/test_statistical_detectors.py

```diff
@@ -111,6 +133,10 @@ def test_run_detection_options(
         else [],
     }
 
+    get_performance_project_settings.return_value = {
+        [project.id]: {"duration_regression_detection_enabled": performance_project_option_enabled}
+    }
```

Comment:
Is there a reason why you have to mock it here instead of just setting the option on the project?

Reply:
no reason. I was copying what we did for other perf issues. I've set the project option directly